src/globals/mixins.scss

```diff
@@ -1275,3 +1100,59 @@
       color: $colour-grey-dark;
   }
 }
+
+$clear-button-brands: (
+  "clear-primary": $colour-primary-blue,
+  "clear-secondary": $colour-secondary-green,
+  "clear-tertiary": $colour-tertiary-coral,
+  "clear-brand-1": $colour-brand-cc-red,
+  "clear-brand-2": $colour-brand-wgmf-purple,
+  "clear-brand-3": $colour-brand-gmf-green,
+  "clear-brand-4": $colour-brand-emf-yellow,
+  "clear-brand-5": $colour-brand-c4c-orange,
+  "clear-brand-6": $colour-brand-grey,
```

Comment:
six is a special case - it's sometimes grey and sometimes turquise so I don't know if we can include it in the loop if we want it to be a pure refactor that doesn't change behaviour. IMHO we should leave 6 as a special case and have the ones that are more regular in the loop.